jwst/resample/resample_utils.py

```diff
@@ -28,7 +28,12 @@ def make_output_wcs(input_models, ref_wcs=None,
     Parameters
     ----------
     input_models : `~jwst.datamodel.ModelLibrary`
-        Each datamodel must have a ~gwcs.WCS object.
+        The datamodels to combine into a single output WCS. Each datamodel must
+        have a ``meta.wcs.s_region`` attribute.
+
+    ref_wcs : gwcs.WCS, None, optional
+        Reference WCS to use as a template for the output WCS. If not provided,
+        the reference WCS will be taken as the WCS of the first input model.
```

Comment:
I think this description is a little unclear - from the code, it looks like ref_wcs is used as the output WCS as is, not as a template to expand on.  Input models are ignored if ref_wcs is specified.